modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -304,7 +304,7 @@
     private boolean needBreadthBar;
     private boolean needLengthBar;
     private boolean tempVisibility = false;
-
+    private boolean suppressBreadthBar;
```

Comment:
Suggestion (optional) : You might want to explicitly initialize this to `false`, since we rely on this for other controls (e.g., `ListView`) which don't call `setSuppressBreadthBar`. (yes, I know Java will initialize it to false anyway, but setting it explicitly could be helpful to a reader of the code)

Reply:
I would argue otherwise: not only it adds visual noise, but any time one steps into the constructor in a debugger, it goes through every field assignment.

Reply:
I don't see it as visual noise (and don't really care about the minor impact on the debugger), but not a big deal either way. I'll re-approve it now.